### ISSUE TITLE
Display Current Layer and Total Layer Count

### ISSUE THREAD

**Issue by mskressin** [BillyBlaze/OctoPrint-FullScreen#1]
Could you please add the layer number being printed along with the total number of layers to the information bar at the bottom of the fullscreen plugin? Such as: Layer 7 of 56. The Print Time Left value is not always very accurate and having the layer information would be very helpful in determining the progress of the print.  Thank you for your consideration and also for the extremely useful  plugin.

**Comment by BillyBlaze** [comment]
Your welcome and thanks for the suggestion.
I think its a good addition.

**Comment by gmccauley** [comment]
I too would love to see this!

**Comment by Fiiti** [comment]
+1

**Comment by BillyBlaze** [comment]
Thanks again for requesting.

After diving into the source of OctoPrint I found out that there is no simple way of getting this information.  As the information does not exist in the core of OctoPrint we need to implement our own analyzing and monitor tools for this information, this tools aren't easily made and require more code then the whole of this plugin. 

I want to keep this plugin small and maintainable and therefor I will not implement or accept such code in this codebase. If there is easy access to this information (such as the temperatures) then I would be more then happy to implement this. That include if another plugin can provide this information.

Closing this request for now.
